src/function-builder.ts

```diff
@@ -127,28 +164,38 @@ export class FunctionBuilder {
    * functions.region('us-east1', 'us-central1')
    */
   region(...regions: Array<typeof SUPPORTED_REGIONS[number]>): FunctionBuilder {
-    if (assertRegionsAreValid(regions)) {
-      this.options.regions = regions;
-      return this;
-    }
+    assertRegionsValidity(regions);
+
+    this.options.regions = regions;
+
+    return this;
   }
 
   /**
    * Configure runtime options for the function.
    * @param runtimeOptions Object with three optional fields:
-   * 1. memory: amount of memory to allocate to the function, possible values
+   * 1. failurePolicy: failure policy policy of the function, boolean `true` is
+   *    equivalent to providing an empty policy.
+   * 2. memory: amount of memory to allocate to the function, possible values
```

Comment:
Let's rephrase here as well to make less awkward: `...to the function, with possible values being '128MB', ...`

Reply:
2e35c26784bb91ca1ee513816cdec440598349be